docs/parallel-workers.md

```diff
@@ -5,8 +5,10 @@ custom_edit_url: https://github.com/stryker-mutator/stryker-js/edit/master/docs/
 
 Stryker will always run checkers and test runners in parallel by creating worker processes (note, not `worker_threads`). The number of such processes forked is determined by the configuration option [`--concurrency`](./configuration.md#concurrency-number). 
 
-However, imagine running these parallel processes on a test suite which uses resources like a database connection, web server or file system. This means these processes can conflict if they write to the same database, file or utilize the same port. 
+However, imagine running these parallel processes on a test suite which uses resources like a database connection, web server or file system. This means these processes can conflict if they write to the same database, file or utilize the same port.
 
+> **Note**
+> Stryker creates a sandbox with a copy of the sources. This directory does not isolate mutants. If the test suite creates a file within the project it will still conflict with other mutations.
```

Comment:
```suggestion
**Note:**
Although Stryker creates a sandbox with a copy of the sources, this directory does not isolate test runners, since all test runners use the same files. If a test suite creates a file within the project directory it will be visible to all other test runners which can have unintended consequences.
```